components/artwork_filter_2/views/filters_view.coffee

```diff
@@ -5,10 +5,15 @@ aggregationsMap = require '../aggregations_map.coffee'
 template = -> require('../templates/filters.jade') arguments...
 
 module.exports = class ArtworkFiltersView extends Backbone.View
+  truncate:
+    museum: 5
```

Comment:
I may have missed this or be reading it wrong, but why is the key for this museum vs institution? I ask because I was looking at the iteration [here](https://github.com/1aurabrown/force/blob/471a97044d593b1eb866a7e0a00eba2853eba3f1/components/artwork_filter_2/templates/filters.jade#L26) and it seems like the key would be institution to match the param from iterating [aggragationsMap](https://github.com/1aurabrown/force/blob/471a97044d593b1eb866a7e0a00eba2853eba3f1/components/artwork_filter_2/aggregations_map.coffee#L17).